### ISSUE TITLE
"panels" buttons should convey appropriate name, state, role, and value

### ISSUE THREAD

**Issue by karlgroves** [jsbin/jsbin#1918]
Buttons located at //*[@id="panels"] 

Buttons' visual labels are

*HTML
*CSS
*JavaScript
*Console
*Output

Activating the buttons is a "toggle" behavior, to show/ hide relevant panels.  For instance, the HTML panel is where you enter HTML, CSS is where you enter CSS, etc.

All of them (except Output) allow input from the user.

The behavior of the buttons is not to navigate. As a consequence, these should be given a `role` attribute value of `button`.

When they are given an "active" state, this should be conveyed programmatically. The converse is also true.  This can be done using aria-label.  For instance:

Current:
`<a class="button group active" href="?css">CSS</a>`

Recommended:
`<a class="button group active" role="button" aria-label="CSS Panel: Active" href="?css">CSS</a>`

Its also likely that the button group should be given a label as well.

I will research this before issuing the resulting PR
